### PR TITLE
Added privateKeyFile update

### PR DESCRIPTION
**Pull request by rosogon**
Just a little help to auto assign the path of the private key in seaclouds.yaml

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -9,3 +9,9 @@ set -u
 echo "Copy additional entities to ${BROOKLYN_HOME}/lib/dropins"
 wget -q -O ${BROOKLYN_HOME}/lib/dropins/brooklyn-modaclouds-0.1.0-SNAPSHOT.jar "http://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=eu.seaclouds-project&a=brooklyn-modaclouds&v=LATEST" --content-disposition
 echo "Copied!"
+
+if [ "$0" = "./setup.sh" ]; then
+    PATTERN="s#\(privateKeyFile:\).*\$#\1 ${PWD}/seaclouds_id_rsa#"
+    sed -i -e "$PATTERN" ./seaclouds.yaml
+    echo "Updated private key path in seaclouds.yaml"
+fi
```
Comment (andreaturli) [inline setup.sh:21]: wdyt about something like

```
DIR=$(cd "$(dirname "$0")" && pwd)
BLUEPRINT=$DIR/seaclouds.yaml
PATTERN="s#\(privateKeyFile:\).*\$#\1 $DIR/seaclouds_id_rsa#"
if [ -e $BLUEPRINT ];then
  sed -i -e "$PATTERN" $BLUEPRINT
  echo "Set 'privateKeyFile: $DIR/seaclouds_id_rsa' in '$BLUEPRINT'";else
  echo "Blueprint $BLUEPRINT doens't exist";
fi
```

?

**Comment by andreaturli** [comment]
@rosogon looks really useful, just a minor suggestion

File: setup.sh (hunk 1 above)
Comment (rosogon) [inline setup.sh:21]: Much more flexible ;)

**Comment by andreaturli** [comment]
@rosogon have you tested it? is it ok? if so, I'll merge it

**Comment by rosogon** [comment]
It works when run $0 is the path to the regular file setup.sh. I mean, not a link or searchable through $PATH.

**Comment by andreaturli** [comment]
do you think we need more?

**Comment by rosogon** [comment]
From my point of view, no. The probability of the script to be run in a non-controlled way are quite low.

**Comment by andreaturli** [comment]
good. merging

**Pull request merged**